print("data_process.py called successfully")